apps/platform-app/src/app/[locale]/(auth)/(routes)/signin/page.module.scss

```diff
@@ -28,6 +28,36 @@
   }
 }
 
+.loginPage {
+  display: flex;
+  justify-content: space-between;
+  gap: 20px;
+
+  margin-block: 33px;
+
+  @include mobile {
+    flex-direction: column;
+  }
+
+  h2 {
+    //styleName: Header 2;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 40px;
```

Comment:
Maybe we should put those styles into global mixins? and use it like this: 
// Define the mixin for Body2 styles 
@mixin body2 { 
 font-size: 14px; 
 font-weight: 400;
 line-height: 20px;
 margin-bottom: 24px;
}
 // Include the mixin in a class 
 .my-class { 
 @include body2; 
 }
 
 what do you think guys?